support/insights-operator/about-insights-operator.adoc

```diff
@@ -0,0 +1,17 @@
+[id="about-insights-operator"]
+= About Insight Operator
+include::modules/common-attributes.adoc[]
+:context: insights-operator
+
+toc::[]
+
+Insights-operator collects anonymized information about:
+
+* The version of the {product-title} cluster and it's components
+* The health, status and current errors reported by {product-title} components and nodes
+* Network topology information
+* The features in use
+
+This information is used by Red Hat to be able to provide better support for the {product-title} cluster. None of the information is shared with third parties.
```

Comment:
We should link to how to turn off the operator here and explain that this is not recommended.

Reply:
We are missing a section on how to turn off the operator.

Reply:
We also need to likely link to other docs sections on how this can be used with a proxy for disconnected situations where the operator wants to send back information still.

Reply:
> We should link to how to turn off the operator here and explain that this is not recommended.

+1 for that!